<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageFilter</class>
 <widget class="QDialog" name="MessageFilter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>125</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>筛选设置</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>重复消息只显示一次</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxDisOnce">
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>十六进制筛选</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radiohexFilter">
         <property name="text">
          <string>RadioButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>显示消息头</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="EditMessagehead"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>不显示消息头</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="EditUnDisplayHead"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>显示消息尾</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="EditMessageTail"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>不显示消息尾</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="EditUnDisplayTail"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="Save">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Close">
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
